src/validation.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len */
+const findErrors = (text, caseName) => {
+  const errors = [];
+
+  if (!text || text === '') {
+    errors.push({ message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+  }
+
+  if (!caseName) {
+    errors.push({ message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+  }
+
+  const possibleCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (caseName && !possibleCases.includes(caseName)) {
+    errors.push({
+      message: `This case is not supported. Available cases: ${possibleCases.join(
+        ', ',
+      ) + '.'}`,
```

Comment:
You could just write it as:
message: `This case is not supported. Available cases: ${possibleCases.join( ', ')}.`,